backend/migrations/2023-02-20-112009_alter_plant_detail/up.sql

```diff
@@ -0,0 +1,5 @@
+-- Your SQL goes here
+ALTER TABLE plant_detail
+ADD COLUMN license VARCHAR;
```

Comment:
It is better if the license is an enum, we don't want any arbitrary licenses. Are we sure that the whole database currently has the same CC license? (Did you scrape this information?)

Reply:
good idea and yes, all the scraped information is available under the same license incl. the data fetched from wikipedia